docs/sdk/client-side/ios/index.md

```diff
@@ -33,6 +35,27 @@ target 'YOUR_TARGET_NAME' do
 end
 ```
 
+</TabItem>
+<TabItem value="spm" label="Swift Package Manager">
+
+You can install the SDK as a dependency in your **Package.swift** file or through Xcode.
+
+```swift showLineNumbers
+// Package.swift
+dependencies: [
+  .package(url: "https://github.com/bucketeer-io/ios-client-sdk.git", .exact("LATEST_VERSION")),
+],
+targets: [
+  .target(
+    name: "YOUR_TARGET_NAME",
+    dependencies: ["Bucketeer"]
```

Comment:
Its usage is deprecated.
For this, please provide a specific type and package name.

```suggestion
    dependencies: [.product(name: "Bucketeer", package: "ios-client-sdk")]),
```